### ISSUE TITLE
fsGroupChangePolicy missing in pod security_context

### ISSUE THREAD

**Issue by kel00s** [hashicorp/terraform-provider-kubernetes#1324]
Hi there,

This is introduced in k8s 1.20 

### Terraform Version, Provider Version and Kubernetes Version
- Finding hashicorp/kubernetes versions matching "~> 2.3.2"...
```
Terraform version: v0.15.5
Kubernetes provider version: 2.3.2
Kubernetes version: 1.20
```

### Terraform Configuration Files
```hcl
# Copy-paste your Terraform configurations here - for large Terraform configs,
# please use a service like Dropbox and share a link to the ZIP file. For
# security, you can also encrypt the files using our GPG public key.
```

### Debug Output
```
╷
│ Error: Unsupported argument
│ 
│   on minio.tf line 163, in resource "kubernetes_stateful_set" "minio":
│  163:           fsGroupChangePolicy = var.security_context.fsGroupChangePolicy
│ 
│ An argument named "fsGroupChangePolicy" is not expected here.
```

### Steps to Reproduce
Put `fsGroupChangePolicy` in pod `variable "security_context`

### Expected Behavior
Should be applied to that block.

### Actual Behavior
Not working

### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

**Comment by kel00s** [comment]
There is a Pull Request to solve this issue. Is there anyone available to review it?

**Comment by kel00s** [comment]
@dak1n1 : could you help us to push forward with the Pull Request and the Issue?

**Comment by sheneska** [comment]
Fixed in #1892
